Jenkinsfile

```diff
@@ -42,6 +42,7 @@ pipeline {
   stages {
```

Comment:
commit message:
Added beforeAgent=true to every skippable stage in all pipelines

At commit description add the link to the new feature: https://issues.jenkins-ci.org/browse/JENKINS-44461 and the reason behind it: If beforeAgent is not set to `true`, the skipped stage agents are in use by Jenkins during the evaluation of the when conditions.